css-gaps-1/Overview.bs

```diff
@@ -482,99 +482,217 @@ Pairing gap intersection points into segments</h4>
 		</figure>
 	</div>
 
-<h3 id="outset">
-Adjusting gap decoration endpoints: The 'column-rule-outset', 'row-rule-outset', and 'rule-outset' properties</h3>
+<h3 id="inset">
+Adjusting gap decoration endpoints: The 'rule-inset' properties</h3>
 
 	<pre class='propdef'>
-		Name: column-rule-outset, row-rule-outset
-		Value: <<length-percentage>>
+		Name: column-rule-edge-start-inset, column-rule-edge-end-inset, row-rule-edge-start-inset, row-rule-edge-end-inset
+		Value: auto | <<length-percentage>>
+		Initial: ''0''
+		Applies to: <a>grid containers</a>, <a>flex containers</a>, <a>multicol containers</a>, and <a>masonry containers</a>
+		Inherited: no
+		Percentages: refer to the <a>crossing gap width</a>
+		Animation type: by computed value type
+	</pre>
+
+	<pre class='propdef'>
+		Name: column-rule-interior-start-inset, column-rule-interior-end-inset, row-rule-interior-start-inset, row-rule-interior-end-inset
+		Value: auto | <<length-percentage>>
 		Initial: ''50%''
 		Applies to: <a>grid containers</a>, <a>flex containers</a>, <a>multicol containers</a>, and <a>masonry containers</a>
 		Inherited: no
 		Percentages: refer to the <a>crossing gap width</a>
 		Animation type: by computed value type
 	</pre>
 
+	For 'edge' properties 'auto' has the following meaning:
+	<dl dfn-for=column-rule-edge-start-inset dfn-type=value>
+		<dt><dfn>auto</dfn>
+		<dd>
+			The endpoint is not offset from the content edge of the container.
+			This is equivalent to a value of ''0''.
+	</dl>
+
+	For 'interior' properties 'auto' has the following meaning:
+	<dl dfn-for=column-rule-interior-start-inset dfn-type=value>
+		<dt><dfn>auto</dfn>
+		<dd>
+			The endpoint is placed halfway into the intersection.
+			This is equivalent to a value of ''50%''.
+	</dl>
+
 	These properties can be used to offset the endpoints of <a>gap decorations</a> relative to the
 	<a>gap intersection points</a> which would normally determine their endpoints.
+
+	<ul>
+		<li>Let |edge-intersection| refer to any [=gap intersection points=] at the content edges of a container.</li>
+		<li>Let |interior-intersection| refer to all other [=gap intersection points=].</li>
+		<li>Let |decoration-segment| refer to any segment that connects a pair of adjacent [=gap intersection points=].</li>
+		<li>Let |start-endpoint| and |end-endpoint| refer to the two endpoints of a given |decoration-segment|.
+		A given |start-endpoint| and |end-endpoint| can be either a |edge-intersection| or a |interior-intersection|.</li>
+	</ul>
```

Comment:
The vertical bar notation is used for variables, typically in an algorithm. (I just found out via the [Bikeshed documentation](https://speced.github.io/bikeshed/#var-and-algorithms) that they're shorthand for `<var>`.)

What we're doing here seems more like defining terms, which would call for `<dfn>`. For example:

```bs
<li>An <dfn>edge gap intersection point</dfn> is a [=gap intersection point=] at the content edges of the container.</li>
```

Then below, I might write:
```bs
The 'column-rule-edge-start-inset' and 'row-rule-edge-start-inset' properties control the behavior for [=edge gap intersection points=],
```
and so on.